prep-javascript-variables/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    var fullName = 'ignacio dominguez';
+    var numberOfPets = '1';
+    var likesVegetables = 'true';
+    var nothing = 'null';
```

Comment:
For numbers, booleans, and null, we'll want to leave off the quotes so that the javascript engine doesn't interpret them as strings